apps/remote_control/lib/lexical/remote_control/code_intelligence/entity.ex

```diff
@@ -465,4 +477,24 @@ defmodule Lexical.RemoteControl.CodeIntelligence.Entity do
       _ -> nil
     end
   end
+
+  defp maybe_local_capture_func(analysis, position) do
+    with {:ok, %Zipper{node: {:/, _, [_, {:__block__, _, _}]}} = zipper} <-
+           Ast.zipper_at(analysis.document, position),
+         %Zipper{node: {:&, _, _}} <- Zipper.up(zipper) do
+      {:/, _, [{local_func_name, meta, _}, {:__block__, _, [arity]}]} = zipper.node
```

Comment:
How can this match? The line right above says the node is `{:&, _, _}`

Reply:
This is the parent node of that node.

Reply:
I see